File: examples/effectsize/example_groups.R
library("effectsize")
data(Boston, package="MASS")
# ANOVA
index  <- sample(nrow(Boston), size=300)
model  <- aov(medv~rad, data=Boston[index,])
# 
es <- eta_squared(model)
es
interpret_eta_squared(es)
#
cohens_f(model)

File: examples/effectsize/example_groups.py
import pandas as pd
import pingouin as pg
Boston = r['MASS::Boston']
# ANOVA
model = pg.anova(data=Boston, dv='medv', between='rad', effsize="n2")
# Partial Eta-squared
es = model['n2']
print('Eta-squared:', es)
# via pingouin
eta2 = pg.compute_effsize(Boston['medv'], Boston['rad'], eftype='eta-square')
print('Eta-squared:', eta2)
# The results in Python differ from R!
